main-site/src/pages/sponsor-us.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import type { HeadFC } from 'gatsby';
+import '../../../shared-ui/style/globals.css';
+import '../../../shared-ui/style/globals.css';
+import Header from '../../../shared-ui/components/header/Header';
+import { mainSiteTabInfo } from '../../../shared-ui/lib/data';
+import Footer from '../../../shared-ui/components/footer/Footer';
+import useMatchMedia from 'react-use-match-media';
+import { min } from '../../../shared-ui/lib/responsive';
+import { StyledPageContainer } from '../../../shared-ui/styled-components/Background.styles';
+import AboutSection from '../../components/about-section/AboutSection';
+import ExploreSection from '../../components/explore-section/ExploreSection';
+import EventsCalendarSection from '../../components/events-calendar-section/EventsCalendarSection';
+import TestimonialsSection from '../../components/testimonials-section/TestimonialsSection';
+
+const SponsorUsPage: React.FC = () => {
```

Comment:
these things are fine as-is at the moment but can we make tech debt ticket to clean up reused state things:
- standardize app-wide state of isDay, isDesktop cleanly thru a useContext or something
- standardize the header / footer inclusion using something like <Outlet />

Reply:
using something like...?

Reply:
<Outlet /> oops